src/rocks_engine.cpp

```diff
@@ -287,19 +288,31 @@ namespace mongo {
                 invariantRocksOK(iter->status());
                 if (prefixIter->Valid() && prefixIter->key().starts_with(prefix)) {
                     // prefix is still alive, let's instruct the compaction filter to clear it up
+                    ++dropped_count;
                     uint32_t int_prefix;
                     bool ok = extractPrefix(prefix, &int_prefix);
                     invariant(ok);
                     {
                         stdx::lock_guard<stdx::mutex> lk(_droppedPrefixesMutex);
                         _droppedPrefixes.insert(int_prefix);
                     }
+                    log() << "compacting dropped prefix: " << int_prefix;
```

Comment:
What log level is used here? This should be debug-only message, it's enough to print that we're compacting some number of dropped prefixes as done below.